packages/automerge-repo-storage-nodefs/src/index.ts

```diff
@@ -4,34 +4,41 @@ import { StorageAdapter } from "@automerge/automerge-repo"
 export class NodeFSStorageAdapter implements StorageAdapter {
   directory: string
 
-  constructor(directory = ".amrg") {
+  constructor(directory = "automerge-repo") {
```

Comment:
prefer if this was called prefix or something, as term is less overloaded, but I don't know what the convention is for storage adapters